documentation/howto/maven_basics.adoc

```diff
@@ -0,0 +1,180 @@
+
+=== Prerequisites
+. You must have an existing Maven project available with the packaging type *pom*. See <<New_Project,Create a New Maven Project>> for instructions to create a new Maven project.
+
+=== Creating a New Maven Module
+. Create a new Maven module as follows:
+.. In the Project Explorer view, right-click the recently created pom project and select menu:New[Project].
+.. From the wizard selection screen, expand `Maven` and select `Maven Module`.
+.. Click `Next >` to continue.
+. Enter the initial module details:
+.. Ensure that the `Create a simple project (skip archetype selection)` check box is not checked. If this check box is selected, the `Select an Archetype` step of the wizard is skipped.
+.. Enter the desired module name in the `Module Name` field. This value corresponds to the Maven project's Project ID.
+.. Use the `Browse` button to locate the desired parent project and select it.
+.. The following optional steps are available for further configuration, if required:
+... (Optional) Check the `Add project(s) to working set` check box to add the newly created projects to a working set.
+... (Optional) Click `Advanced` to view additional optional advanced configuration for the new Maven project, such as:
+.... `Resolve Workspace projects`: dependencies opened as workspace projects will be resolved without having to install them to your local Maven repository first. This way, any changes made to one of these dependencies will have an immediate effect on other projects consuming it (compilation, refactoring, etc.). When `Resolve Workspace projects` is disabled, dependencies existing in the workspace must be installed to your local Maven repository after any change (by running `mvn install`), in order to see effects in projects consuming them.
+.... `Profiles`: select a set of Maven profiles to activate or deactivate in the workspace. Profiles are defined in the project `pom.xml`, or inherited from a parent `pom.xml`, or defined in the relevant `settings.xml`.
+.... `Name templates`: allows you to disambiguate projects names in the workspace by prepending or appending the group ID or SCM branch names to the default artifact ID.
++
+.Set the Module Name and Parent
+image::mvnbasic_select_module_name.png[Set the Module Name and Parent]
++
+.. When the configuration is complete, click `Next` to continue.
+. Enter the module archetype information:
+.. Ensure that the `Show the last version of Archetype only` check box is checked. This ensures that only the latest version of each archetype displays.
+.. Select an archetype based on the purpose of the project you are creating. Use the keyword *maven-archetype-quickstart* in the `Filter` field to locate a sample Maven project archetype.
++
+.Select a Module Archetype
+image::mvnbasic_module_archetype.png[Select a Module Archetype]
++
+.. Click Next to continue.
```

Comment:
beware, the following instructions pertain to the "create a simple project" path. You should just have to set/override the groupId, version, package and archetype properties if necessary